commands/run.go

```diff
@@ -0,0 +1,219 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/chasinglogic/redalert/checks"
+	"github.com/chasinglogic/redalert/testfile"
+	"github.com/spf13/cobra"
+	yaml "gopkg.in/yaml.v2"
+)
+
+type multiFlags []string
+
+func (sf *multiFlags) String() string {
+	return strings.Join(*sf, ", ")
+}
+
+func (sf *multiFlags) Set(value string) error {
+	*sf = append(*sf, value)
+	return nil
+}
+
+func (sf *multiFlags) Type() string {
+	return "string"
+}
+
+var (
+	fileFlag string
+	suites   multiFlags
+	tests    multiFlags
+	output   string
+)
+
+func init() {
+	Run.Flags().StringVarP(&fileFlag, "file", "f", "", "Path to test file to run. Note: this overrides default test file detection.")
+	Run.Flags().StringVarP(&output, "output", "o", "", "Output format to use. Valid values: text, json, csv. Default: text")
+	Run.Flags().Var(&suites, "suite", "Suite or alias name to run, can be passed multiple times.")
+	Run.Flags().Var(&tests, "test", "Specific test name to run, can be passed multiple times.")
+}
+
+// Run will simply run the tests. It takes zero to one arguments. If no arguments
+// are given it looks for a tests.(yml|yaml) or redalert.(yml|yaml) in the local
+// directory, in `$HOME/.redalert/` (`%APPDATA%` instead of `$HOME` on windows.)
```

Comment:
// directory, in `$HOME/.redalert/` (`%APPDATA%\redalert\` on windows.)

Might be more accurate.